docs/output/accounts.md

```diff
@@ -609,7 +609,7 @@ Describes customer support contact information for a business account.
 
 | Property | Type | Description |
 | ---- | ---- | ----------- |
-  | countries | `Array.<string>`|  |
+  | countries | `Array.<string>`| Countries of operation for an account |
```

Comment:
Not entirely sure, but this is for business accounts, right? 

If so maybe we can put "Countries where a business operates"? Since the current description is a little unclear.

Reply:
The country API endpoints don't specify, therefore I read it as it's for any account. If it is only for business accounts then the API needs to be updated.